content/api/entity-reference/gift/index.html

```diff
@@ -123,12 +123,12 @@
             <tr>
                 <td>gift_aid_amount</td>
                 <td><a href="{{ stache.config.constituent_entity_reference }}#Currency">currency</a></td>
-                <td>The amount of tax reclaimed from Gift Aid for this gift. For UK only.</td>
+                <td>The amount of tax reclaimed from Gift Aid for this gift.  For UK formatted databases only. This is a computed field based on total qualified gift aid amount across all <code>gift_splits</code>.</td>
             </tr>
             <tr>
                 <td>gift_aid_qualification_status</td>
                 <td>string</td>
-                <td>The Gift Aid qualification status of the gift. Available values are: qualified, notqualified, and partlyqualified. For UK only.</td>
+                <td>The Gift Aid qualification status of the gift. Available values are: qualified, notqualified, and partlyqualified. For UK formatted databases only. This is a computed field based on tax declaration information and database format.</td>
```

Comment:
The available values should be italicized to be consistent with how we list available values elsewhere.

Same note as above about adding "formatted databases." My 2 cents is that we should stick with "For the UK only" (including the missing "the"). But if we want to add "formatted databases," I'd do that in all instances, not just this one. And I'd hyphenate "UK-formatted."

I'd also rework the last sentence to avoid the passive "based on" formation. I'd change it to: "This computed field determines the qualification status based on tax declaration information and the database format."